plugins/terraform/layer0/resource_layer0_service.go

```diff
@@ -92,6 +104,7 @@ func resourceLayer0ServiceRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("environment", service.EnvironmentID)
 	d.Set("load_balancer", service.LoadBalancerID)
 	d.Set("scale", service.DesiredCount)
+	d.Set("stateful", bool(service.ServiceType == models.DeployCompatibilityStateful))
```

Comment:
[nit] ditto

Reply:
Is now `d.Set("stateful", service.Stateful)`. ✅